homework-16_DataManipulation.sql

```diff
@@ -0,0 +1,31 @@
+DELETE FROM "RESERVATIONS";
+DELETE FROM "ROOMS";
+DELETE FROM "GUESTS";
+DELETE FROM "HOSTS";
+DELETE FROM "USERS";
+
+INSERT INTO "USERS" (id, name) VALUES (1, 'Host and Guest 1');
+INSERT INTO "USERS" (id, name) VALUES (2, 'Guest 2');
+INSERT INTO "USERS" (id, name) VALUES (3, 'Host and Guest 3');
+
```

Comment:
Please add comments on what you meant with each of the inserts below.

Reply:
Done.